index.js

```diff
@@ -115,6 +115,10 @@ function createBtns(totalPage, current) {
     }
 }
 
+window.onpopstate = (event) =>{
+    window.location.reload();
```

Comment:
Reloading the entire page serves the purpose as of now, but it looks more like a hack, or a workaround, rather than a solution.
Is there any way you can fetch content for the required page and render them instead of reloading the entire page? (Maybe you can try playing around wit the `renderSolutions()` function)